Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -207,13 +227,15 @@ extension Channel {
                 )
             }
         } else {
+            let loopBoundStreamDelegate = NIOLoopBound(streamDelegate, eventLoop: self.eventLoop)
+            let loopBoundPosition = NIOLoopBound(position, eventLoop: self.eventLoop)
```

Comment:
We know we're not on the EL here: these will precondition failure on init.

Reply:
I've made it a requirement that the `streamDelegate` be `Sendable` and left the position warning there.